public/app/plugins/datasource/loki/querybuilder/parsing.ts

```diff
@@ -686,3 +686,19 @@ function handleKeepFilter(expr: string, node: SyntaxNode, context: Context): Que
     params: labels,
   };
 }
+
+export function makeBinOp(
```

Comment:
This can be deleted and we can go back to using the same one as prom now that they have the same ordering of properties.